tests/execution_engine/test_sqlalchemy_batch_data.py

```diff
@@ -134,3 +135,16 @@ def test_instantiation_with_and_without_temp_table(sqlite_view_engine, sa):
     )
     res = execution_engine.get_batch_data_and_markers(batch_spec=my_batch_spec)
     assert len(res) == 2
+
+
+def test_instantiation_with_unknown_dialect(sqlite_view_engine):
```

Comment:
I think unsupported is the wrong word here; "other" would suffice.

Reply:
Done!